src/negotiate.rs

```diff
@@ -481,6 +481,23 @@ impl SspiImpl for Negotiate {
             }
         }
 
+        #[cfg(feature = "scard")]
+        if let Some(Some(CredentialsBuffers::SmartCard(identity))) = builder.credentials_handle {
```

Comment:
I improved security package negotiation. In the case of the smart card logon, we should always use the Kerberos protocol